docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: "3.8"
```

Comment:
It will raise warning for newer version of docker compose (In my case 2.27.1)
`docker-compose.yml: version is obsolete`